src/main/sensors/compass.h

```diff
@@ -17,6 +17,10 @@
 
 #pragma once
 
+#include "io/escservo.h"
+
+#include "flight/navigation_rewrite.h"
```

Comment:
Is it okay to have the includes here? I get this error message when they are in compass.c
> ./src/main/sensors/compass.h:35:89: error: unknown type name 'escAndServoConfig_t'
> ./src/main/sensors/compass.h:35:89: error: unknown type name 'escAndServoConfig_t'